ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/tacacs-servers.j2

```diff
@@ -7,6 +7,10 @@
 | {% if host['vrf'] is defined and host['vrf'] is not none %} {{ host['vrf'] }} {% else %} Global Routing Table {% endif %}| {{ host['host'] }} |
 {%  endfor %}
 
+{%    if tacacs_servers.policy_unknown_mandatory_attribute_ignore is defined and tacacs_servers.policy_unknown_mandatory_attribute_ignore == true %}
+Policy unknown-mandatory-attribute ignore is configured
+{%    endif %}
+
```

Comment:
The blank line should be enclosed to the `if` block to not append an extra blank line when code is not rendered.

Reply:
Thanks done!